packages/strapi-plugin-content-manager/admin/src/components/InputUID/Options/index.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import pluginId from '../../../pluginId';
+import OptionsWrapper from './wrapper';
+import Option from './Option';
+import OptionsTitle from './OptionsTitle';
+import RightOptionLabel from './RightOptionLabel';
+
+const Options = ({ options, title }) => {
+  const [displayRightOptionLabel, setDisplayRightOptionLabel] = useState(false);
+  const handleOptionMouseEnter = () => {
+    setDisplayRightOptionLabel(true);
+  };
+  const handleOptionMouseLeave = () => {
+    setDisplayRightOptionLabel(false);
+  };
+
+  return (
+    <OptionsWrapper>
+      {title && <OptionsTitle>{title}</OptionsTitle>}
+      {options.map(option => (
+        <Option
+          onMouseEnter={handleOptionMouseEnter}
```

Comment:
Maybe you can handle the display of the option label using css instead of using full javascript here.